octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/MultiSelectComboBox.java

```diff
@@ -240,6 +244,15 @@ public void setSelected(Collection<T> selectOptions, boolean fireSelectionHandle
 		for (T selectOption : selectOptions) {
 			if (options.contains(selectOption)) {
 				setSelected(selectOption, true, false);
+			} else {
+	            ILog log = Activator.getDefault().getLog();
```

Comment:
Indentation in the file is ok here, github shows it badly.

Reply:
dunno, I'll check

Reply:
![image](https://user-images.githubusercontent.com/23213115/39228352-6294d68a-4866-11e8-9719-0cc2f1a9028d.png)

looks fine here